.travis.yml

```diff
@@ -7,8 +7,8 @@ services:
 - redis-server
 env:
   global:
-  - secure: cfX/Vagk7SskUp1cnuTTfQHNWz6pjx7bHJOa25ahk/3Jvsqz37qf8MLNs8uRUb/LN63G/YNUtrcRZQiyDS9WHehQy0p7IO68UuOG4zqkJE0iemR0kToCTHMbyLTOB70nhTuwmRfh1KeT5Ja3D9TOtApQErU5IQaKm5jP8CZb4PA=
-  - secure: Vhb4qkFc0BcRkJaPU6ZbM9vPHaEHoz/A1LlQ1IMrEdXS/nUabd4l32q6zjJZvfHJ5bI1PXgk1BFztVGT65UT7Uu17KKawzLZWDLmrgzRESkKDj/Ld9Wg79DY6ooJ/juWBrWuJtoNOFKy6CqLdCKWIzNkd9zG+au3qoFP8LfhRh4=
-  - secure: dKmkzv90Fn1JQpSc2l+cESLFYw3bnN5TsgDKTc2tFFi285EJuqd2z22wfDGcaYAoqX/ck+YTdPqVH6trrC7b3pSMqJSBEaHtWiYMbEkmf/QWrVAHy4LEKFAcs0oACjwVRe7CGBu8RRBWcl8I1STpYm11S8WDe8Y/GHo0HnXipHo=
-  - secure: FejMHUo69w52zjPBMfUThhhgj7md/dj+7gf+1d0sgDuDfZ+gECEyHF8iWvkm64k3j+PX4E/y44HdV1+qIMWTGNA0xSZNFEHBBJrEC1BRKm9BkT8W0KBcupmLRHW0BtLWMrqtKOgOyDEdoufuLx7wDpTNslPeGJ4NUZIGEDTuwLc=
-  - secure: DyEw82o0ozA9rbbgYUk8jiC/KNwB0x0OuVv/TX7Xso3uXgfTL1m3IWYLDX6kvV4QIvHNaNdj1g02ddnRwZwuvA3IH81J8+mkZ6aIXTS1YKZPEZMSqewZNOGQPRVaUOeQ9ymQtE7bfxWkViF2JzJlGzc3IxYUKHeo+NE/Hgehuz4=
+  - secure: LD0/aeZ4yCodxjrOQgYJYZzuoHw0MqoRAq5V6mkmcFbq9D2E1a9nE8dGv4IxFoAyY4Zg5r6uTAtTd5d4L09vT3H0KHnO59NEktUjRMFu24yal83g8Y4U+aaj4NAWebaioq9VVMVlmBrnJKtV972ApzgeslazqEW3u+z0HOBS8ik=
+  - secure: XUGLTP7rqOxYXZMob6Uqc4uoUpIypP3DO+U6JUChZHLsCqM3SROnGDiaUOqusWoR4a8XxE6S2GCzXYa80hfCof1vtWfcEdPZSwMF4Ue/xR1LR4JTBTayOmJTTSnR6e29MhSPK/nIVXbDTeJsY8ctCSvZFOyrOTp6k1xyBcbKEDI=
+  - secure: acqb/Dua5RNOUEOqU4fVIg2gd93OJW9j26gT5QhFFHFSXSdTDQoP++O7yc3tPyzvopG96BnNSRJNfolskiDXy8/z9+IJpBhGEpYMmSceHBOSOCYCvsdpYBCRjgWya8Di1THCNQV0B4hBKilR0rnlww58zx65TYY2Y2c0eB5ToV4=
+  - secure: Mcv5fU7hpiQBZ2KHr0gevHpvTiqIGNIfKa4vyUawlbiL5vnByWudlJLSd2e+Vz9JwP3K0k+4p/PeIUn6W52Nisn6KEvTJfGKxnL4IT4ea86xnSeKQWScQW6QrKWI9+gPmrIT5bMM82mzEt+fvA6aAjRl/diq7iJw5xKjclntQAw=
+  - secure: WG9Kp4V3DfSqvmOV+FlwQtYlCOWUDKaC9XMpTZUd1rxLNSlVpHbXSLwJXhDcY4HHdD4A9eHDhckIdc+IpQwygwI6cBscfZNBVdq/w3Yjhujh8qLDR1dX4fCrCL3jMOMyrkZ2g8i/dBNtLvIwYA+sRikVqNztovbWFQh6oAYqVgA=
```

Comment:
There are currently a lot of tests that need a directory with verification workflow enabled
https://github.com/stormpath/stormpath-sdk-ruby/blob/master/spec/spec_helper.rb#L57

the tests where failing because this directory was probably accidentally deleted when the tests where running. Since we don’t have an ability to create this kind of directory by ruby itself it needs to be created “manually”.

Taking in the consideration I’m the only one working oh the project and the time difference between us I temporarily just linked the tests to my account, this way I have more control. 

If this is a problem we can revert.